grouped-categories.js

```diff
@@ -434,9 +440,13 @@
 
 				label = chart.renderer.text(name, 0, 0, useHTML)
 					.attr(mergedAttrs)
-					.css(mergedCSS)
 					.add(axis.labelGroup);
 
+				//css should only be set for non styledMode configuration. #167
+				if (label && !isStyledMode(chart)) {
```

Comment:
There's no need to check for `Highcharts.version`, simply check if `chart.options.styledMode !== true` (old versions used to have `styledMode` undefined, so it's fine).

Reply:
yes it makes sense, removed highcharts version check. please verify.